packages/backend/migrations/20231121184537_add_resource_fkeys_webhook_event.js

```diff
@@ -0,0 +1,96 @@
+exports.up = function (knex) {
+  return knex.schema
+    .table('webhookEvents', function (table) {
+      table
+        .uuid('outgoingPaymentId')
+        .nullable()
+        .references('outgoingPayments.id')
+        .index()
+        .onDelete('CASCADE')
+      table
+        .uuid('incomingPaymentId')
+        .nullable()
+        .references('incomingPayments.id')
+        .index()
+        .onDelete('CASCADE')
+      table
+        .uuid('walletAddressId')
+        .nullable()
+        .references('walletAddresses.id')
+        .index()
+        .onDelete('CASCADE')
+      table
+        .uuid('peerId')
+        .nullable()
+        .references('peers.id')
+        .index()
+        .onDelete('CASCADE')
+      table
+        .uuid('assetId')
+        .nullable()
+        .references('assets.id')
+        .index()
+        .onDelete('CASCADE')
```

Comment:
So basically, for every webhook event concerning liquidity (or all of them?) we store a reference to that resource?

Reply:
> So basically, for every webhook event concerning liquidity (or all of them?) we store a reference to that resource?

Yes, all of them. Each webhook has a related resource, except for the `WalletAddressEventType.WalletAddressNotFound` (because there is no wallet address).

I was mostly concerned with outgoing payment (only one used ATM) but added all of them so that they aren't just hiding in the json data payload.

Going forward, this mean a webhook will need to reference at least one of these events, or we update the constraint (to reference a different resource, or carve out an exception like this migration does for the wallet address not found event).